### PR TITLE
Prevent creation of multiple parsec orgs at a time

### LINKED ISSUE #3698: AssertionError
Sentry Issue: [PARSEC-R2M](https://sentry.io/organizations/scille/issues/3784781277/?referrer=github_integration)

```
TimeoutError: [Errno 110] Connection timed out
(5 additional frame(s) were not displayed)
...
  File "http/client.py", line 952, in send
    self.connect()
  File "http/client.py", line 1419, in connect
    super().connect()
  File "http/client.py", line 923, in connect
    self.sock = self._create_connection(
  File "socket.py", line 843, in create_connection
    raise err
  File "socket.py", line 831, in create_connection
    sock.connect(sa)

URLError: <urlopen error [Errno 110] Connection timed out>
(9 additional frame(s) were not displayed)
...
  File "urllib/request.py", line 517, in open
    response = self._open(req, data)
  File "urllib/request.py", line 534, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "urllib/request.py", line 1389, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "urllib/request.py", line 1349, in do_open
    raise URLError(err)

BackendNotAvailable: <urlopen error [Errno 110] Connection timed out>
  File "parsec/core/gui/create_org_widget.py", line 59, in _do_create_org
    new_device = await bootstrap_organization(
  File "parsec/core/invite/organization.py", line 102, in bootstrap_organization
    rep = await failsafe_organization_bootstrap(
  File "parsec/core/invite/organization.py", line 127, in failsafe_organization_bootstrap
    rep = await cmd_organization_bootstrap(
  File "parsec/core/backend_connection/anonymous.py", line 126, in organization_bootstrap
    return await _anonymous_cmd(
  File "parsec/core/backend_connection/anonymous.py", line 63, in _anonymous_cmd
    raise BackendNotAvailable(exc) from exc

JobResultError: ('connection-error', {'exc': BackendNotAvailable(URLError(TimeoutError(110, 'Connection timed out'))), 'origin': None})
  File "parsec/core/gui/trio_jobs.py", line 122, in __call__
    result = await cast(Awaitable[R], self._fn(*self._args, **self._kwargs))
  File "parsec/core/gui/create_org_widget.py", line 75, in _do_create_org
    raise JobResultError("connection-error", exc=exc)

AssertionError: 
  File "parsec/core/gui/create_org_widget.py", line 304, in _on_req_error
    assert self.create_job is job

Exception in Qt slot
```

### PR DESCRIPTION
**Pull request by ADcorpo**
# What has changed ?

Prevent creating multiple orgs simultaneously from parsec's GUI

- [x] Does this PR affect the User ?

closes #3698

### CODE REVIEW

**Comment by ADcorpo** [comment]
This currently does not pass tests, because we unplug the org creation widget from Qt signals. The problem is our tests rely on them to know if an organization was successfully created.

File: parsec/core/gui/create_org_widget.py [hunk 1]
```diff
@@ -55,28 +50,13 @@ async def _do_create_org(
...
-        raise JobResultError("out-of-ballpark", exc=exc)
-    except InviteError as exc:
-        raise JobResultError("invite-error", exc=exc)
+    new_device = await bootstrap_organization(
+        backend_addr, human_handle=human_handle, device_label=device_label
+    )
+    # The organization is brand new, of course there is no existing
+    # remote user manifest, hence our placeholder is non-speculative.
+    await user_storage_non_speculative_init(data_base_dir=config.data_base_dir, device=new_device)
+    return new_device
```
Comment (vxgmichel) [inline parsec/core/gui/create_org_widget.py:59]: I don't think `_do_create_org` make sense as function anymore, those two calls can be moved directly in the async event handler.

**Review by vxgmichel — APPROVED** [review]
A single comment but LGTM otherwise :+1:

File: parsec/core/gui/create_org_widget.py [hunk 2]
```diff
@@ -278,73 +314,6 @@ async def _on_next_clicked(self) -> None:
...
-            else:
-                err_msg = _("TEXT_ORG_WIZARD_ORGANIZATION_ALREADY_EXISTS")
-        elif status == "connection-refused":
-            err_msg = _("TEXT_ORG_WIZARD_CONNECTION_REFUSED")
-        elif status == "connection-error":
-            err_msg = _("TEXT_ORG_WIZARD_CONNECTION_ERROR")
-        elif status == "out-of-ballpark":
-            err_msg = _("TEXT_BACKEND_STATE_DESYNC")
-        else:
-            err_msg = _("TEXT_ORG_WIZARD_UNKNOWN_FAILURE")
```
[the remark this answers is not in the document]
Comment (vxgmichel) [reply parsec/core/gui/create_org_widget.py:336]: It's ok, I double checked all the status

File: parsec/core/gui/create_org_widget.py (hunk 1 above)
Comment (ADcorpo) [reply parsec/core/gui/create_org_widget.py]: fixed in 0eb544c49a90d6041bcc7f707f3ff8cfa59bbf48

**Pull request merged**